isthmus/src/main/java/io/substrait/isthmus/SubstraitRelVisitor.java

```diff
@@ -207,22 +207,9 @@ public Rel visit(org.apache.calcite.rel.core.Correlate correlate) {
     // right input of correlated-join is similar to a correlated sub-query
     apply(correlate.getRight());
 
-    var joinType = asJoinType(correlate);
```

Comment:
I wonder if this call was intended to have a side-effect of throwing an exception if the join type was not INNER or LEFT? I guess if the tests still pass then we can assume it wasn't intended behaviour though.

Reply:
Good observation though I think it does not make a huge difference since currently it calls `return super.visit(correlate)` in the next line which then calls `visitOther()` which throws an `UnsupportedOperationException` for `org.apache.calcite.rel.core.Correlate`. So this is effectively replacing an `IllegalArgumentException` with an `UnsupportedOperationException`.